spec/oas3.fetched/paths/organizations/id/devices.yaml

```diff
@@ -21,6 +21,8 @@ get:
           - compute
           - storage
           - vmce
+          - legacy_gen
```

Comment:
I wonder if this is a second filter, a subset of compute, or something else 🤔. Would it indicate a subset of older or newer plans for any `line`?